docs/no-legacy-template-syntax.md

```diff
@@ -13,14 +13,16 @@ The following patterns are considered warnings:
 html`<x-foo bar?=${x}>`;
 html`<x-foo on-bar=${x}>`;
 html`<x-foo bar$=${x}>`;
+html`<x-foo bar=${x}>`;
```

Comment:
this is valid syntax in both legacy and non-legacy (it sets an attribute in current lit). so we probably don't need this line?

Reply:
I was aligning the before with the after.

before attribute was `bar$=${x}`
after attribute is `bar=${x}`